docs/getting-started-scalardb.md

```diff
@@ -1,4 +1,12 @@
-# Getting Started with Helm Charts (ScalarDB Server)
+# [Deprecated] Getting Started with Helm Charts (ScalarDB Server)
+
+{% capture notice--info %}
+**Note**
+
+ScalarDB Server is now deprecated. Please use [ScalarDB Cluster](https://github.com/scalar-labs/scalardb-cluster/blob/main/docs/setup-scalardb-cluster-on-kubernetes-by-using-helm-chart.md) instead of it.
```

Comment:
Just a minor suggestion here. (Changes made here are consistent across other docs in this PR.)

```suggestion
ScalarDB Server is now deprecated. Please use [ScalarDB Cluster](https://github.com/scalar-labs/scalardb-cluster/blob/main/docs/setup-scalardb-cluster-on-kubernetes-by-using-helm-chart.md) instead.
```